rules/opentitan_gdb_test.bzl

```diff
@@ -0,0 +1,125 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+load("@lowrisc_opentitan//rules:rv.bzl", "rv_rule")
+
+def _opentitan_gdb_fpga_cw310_test(ctx):
+    test_script = '''#!/usr/bin/env bash
+        set -e
+
+        # Do not mask failures in the left-hand side of pipelines.
+        set -o pipefail
+
+        COLOR_RED='\\x1b[0;31m'
+        COLOR_GREEN='\\x1b[0;32m'
+        COLOR_PURPLE='\\x1b[0;35m'
+
+        function prefix_lines() {{
+            LABEL="$1"
+            COLOR="$2"
+            COLOR_RESET='\\x1b[m'
+            sed -Eu "s/(.*)/$COLOR[$LABEL]$COLOR_RESET \\1/"
+        }}
+
+        set -x
+
+        ./opentitantool load-bitstream --rom-kind=TestRom rom.bit
```

Comment:
Sorry for the late comment: these invocations of `opentitantool` should really include `--rcfile= --interface=cw310`.  (that is `--rcfile=<empty string>`).

Without `--rcfile=`, opentitantool will parse the user's `$HOME/.config/opentitantool/config`.
The explicit `--interface=cw310` is required because without a local config file, opentitantool defaults to the "empty" transport.

Reply:
Already done in PR #15024 :)